### PR TITLE
REL-4743 Stress testing after the implementation of bulk task operations.

### PR DESCRIPTION
**Pull request by dkhoroshylovxl**

### CODE REVIEW

File: runner/src/main/scala/stress/utils/Scenarios.scala [hunk 1]
```diff
@@ -104,6 +104,23 @@ object Scenarios {
       )
   }
 
+  def bulkOpsScenario(repeats: Int): ScenarioBuilder = {
+    scenario("(Bulk) Ops person")
+      .repeat(repeats)(
+        exec(Releases.getRandomTreeRelease)
```
Comment (hilic) [inline runner/src/main/scala/stress/utils/Scenarios.scala]: Not a 100% certain but this means that you depend on having dependency trees generated, which are used for the dependencies scenario? So without reading the source code I have no idea that the two are correlated, so maybe update the read me or reuse the active releases (which are generated by default) or something...

File: runner/src/main/scala/stress/utils/Scenarios.scala [hunk 2]
```diff
@@ -104,6 +104,23 @@ object Scenarios {
       )
   }
 
+  def bulkOpsScenario(repeats: Int): ScenarioBuilder = {
+    scenario("(Bulk) Ops person")
```
Comment (hilic) [inline runner/src/main/scala/stress/utils/Scenarios.scala]: `Ops person (bulk operations)`?

File: runner/src/main/resources/runner.conf [hunk 3]
```diff
@@ -21,6 +21,9 @@ xl {
       ops = 5
       ops = ${?ops}
 
+      bulkOps = 5
```
Comment (hilic) [inline runner/src/main/resources/runner.conf]: Document these in the README please

File: runner/src/main/scala/stress/chain/Tasks.scala [hunk 4]
```diff
@@ -66,6 +66,42 @@ object Tasks {
...
+           |"team":"Release Admin", "owner": "admin"}""".stripMargin)
+    })
+    .exec(
+      http("Change assignment on tasks")
+        .put("/tasks/reassign")
+        .body(StringBody("${changeAssignmentOnTasksBody}"))
+        .asJSON
+    )
+
+  def removeTasks(): ChainBuilder = exec(session => {
```
Comment (hilic) [inline runner/src/main/scala/stress/chain/Tasks.scala]: If I'm not mistaken, most scenarios don't actually delete stuff from a release, but query stuff or edit, so if you invoke this are you sure that after a while you won't delete all available tasks and that this scenario will be useless?

File: runner/src/main/scala/stress/chain/Tasks.scala [hunk 5]
```diff
@@ -40,7 +40,7 @@ object Tasks {
 
   def openAndPoll(httpName: String, filter: String, duration: Duration, taskPollPause: Duration): ChainBuilder = open(httpName, filter)
     .exec(session => {
-      session.set("pollTasksBody", s"""{"ids":[${session.taskIds.map(s => s""""$s"""").mkString(",")}]}""")
+      session.set("pollTasksBody", s"""{"ids":[${session.taskIds.map(s => s""""$s"""").mkString(", ")}]}""")
```
Comment (dkhoroshylovxl) [inline runner/src/main/scala/stress/chain/Tasks.scala]: Just a typo. Removed

File: runner/src/main/scala/stress/chain/Tasks.scala (hunk 4 above)
Comment (dkhoroshylovxl) [inline runner/src/main/scala/stress/chain/Tasks.scala]: Yes, I know about that issue. I have asked Bulat regarding the case. You have to run generators before each schenario :) Maybe this part needs to be improved later but definitely out of scope my story.

File: runner/src/main/scala/stress/simulations.scala [hunk 6]
```diff
@@ -113,6 +114,11 @@ class ReleaseManagerSimulation extends SimulationBase(releaseManagerScenario(1))
 class OpsSimulation extends SimulationBase(opsScenario(1))
 
 /**
+  * X ops people are working with bulk operations in XL Release
+  */
+class BulkOpsSimulation extends SimulationBase(bulkOpsScenario(1))
```
Comment (dkhoroshylovxl) [inline runner/src/main/scala/stress/simulations.scala]: How do you like OpsBulk? :)

File: runner/src/main/scala/stress/utils/Scenarios.scala (hunk 2 above)
Comment (dkhoroshylovxl) [inline runner/src/main/scala/stress/utils/Scenarios.scala]: Yep, make sense. Fixed.

File: runner/src/main/scala/stress/utils/Scenarios.scala (hunk 1 above)
Comment (dkhoroshylovxl) [inline runner/src/main/scala/stress/utils/Scenarios.scala]: Just take a look on any other scenario, it works the same way.. I don't like the idea, but I am following the approach here. Should we create a ticket to fix it?

File: runner/src/main/scala/stress/simulations.scala (hunk 6 above)
Comment (hilic) [inline runner/src/main/scala/stress/simulations.scala]: Better I guess 😄

File: runner/src/main/scala/stress/utils/Scenarios.scala (hunk 1 above)
Comment (hilic) [inline runner/src/main/scala/stress/utils/Scenarios.scala]: Hmm, not really, the dependency scenario is the only one that uses that call I think... and when running a dependency scenario, you're gonna create a dependency tree?

**Comment by dkhoroshylovxl** [comment]
Hrvoje, please review the new changes

**Comment by hilic** [comment]
@dkhoroshylovxl LGTM!

**Pull request merged**